srfi-178.html

```diff
@@ -104,15 +105,15 @@ <h3>Notation</h3>
 
       <dt><i>vec</i></dt>
       <dd>
-        A heterogeneous vector; that is, it must
+        An heterogeneous vector; that is, it must

```

Comment:
"an heterogeneous" is not correct, at least in American English.

Reply:
Thanks; reverted that hunk.